robottelo/cli/content_export.py

```diff
@@ -39,36 +39,36 @@ def list(cls, options=None):
         return cls.execute(cls._construct_command(options), output_format='json')
 
     @classmethod
-    def completeLibrary(cls, options):
+    def completeLibrary(cls, options, timeout=None):
```

Comment:
I'd prefer setting some long timeout to setting no timeout at all. I'd also think about taking this value from config by default - there are some timeout related settings. Applies to all instances of this.

Reply:
I would set the timeout for each and every function here. I would rather see it set in in the `execute` method.
If there is no timeout specified in the `execute` function call in `robottelo/cli/content_export.py`, it would use the default timeout value from the config.

Reply:
Isn't that the current behavior?

Reply:
Disregard what I wrote above. 
We already have it implemented here:
https://github.com/SatelliteQE/robottelo/blob/99b70756a965db1af0dfb770814033d71c8cb774/robottelo/hosts.py#L102-L104

So every command that is executed on the host has assigned the timeout value from the robottelo config.

If @rmynar wants to use the timeout from the robottelo config, he has to use `None`. If he wants to have no timeout for the command he should use `0`. And of course there might be timeout explicitly defined. We can specify it in time units. See the broker helper:

https://github.com/SatelliteQE/broker/blob/da7c35f24f0d44d262268e8b6988e7f250ba012e/broker/helpers.py#L372-L391

The question is whether we want to set the timeout here in the `completeLibrary` function or leave it up to the test that is using the function.

Reply:
The problematics of setting timeout was described by @ogajduse.
I would like to use timeout as non-mandatory (by default use value from settings) and increase it only in specific tests.